cocotb/__init__.py

```diff
@@ -112,13 +113,15 @@ def _reopen_stream_with_buffering(stream_name):
 # FIXME is this really required?
 _rlock = threading.RLock()
 
+LANGUAGE = os.getenv("TOPLEVEL_LANG")
+
 
 def mem_debug(port):
     import cocotb.memdebug
     cocotb.memdebug.start(port)
 
 
-def _initialise_testbench(root_name):
+def _initialise_testbench(argv_):
```

Comment:
Opened https://github.com/cocotb/cocotb/issues/1428 to discuss just putting this in `sys.argv`, which I think we touched on before.